plugins/experimental/traffic_dump/session_data.cc

```diff
@@ -69,6 +69,12 @@ std::unordered_map<std::string, std::string> tag_to_node = {
   {std::string(IP_PROTO_TAG_HTTP_3), R"("name":"http","version":"3")"},
 };
 
+std::unordered_map<std::string, std::string> http_tag_to_version = {
+  {std::string(IP_PROTO_TAG_HTTP_0_9), "0.9"},  {std::string(IP_PROTO_TAG_HTTP_1_0), "1.0"},
```

Comment:
Why make the key a `std::string`? All of the keys are literals so `std::string_view` would work as well.